### ISSUE TITLE
POM is repeating plugin settings from parent

### ISSUE THREAD

**Issue by ralphlange** [epics-base/jca#20]
The POM is repeating plugin settings that it inherits from the parent POM.
This should be avoided as it defeats the original purpose of using a parent POM, making sure that versions of dependencies and plugins are consistent throughout a multi-module project.

**Comment by shroffk** [comment]
Yes you are absolutely right...but in this case, due to the rather peculiar behaviour of the project with respect to tests/releases/git commits. I was playing around with the duplicate settings/definitions.

We should remove all duplicate parts before the next release.
